docs/actions/add_git_tag.md

```diff
@@ -53,18 +55,20 @@ add_git_tag(
 
 ## Parameters
 
-Key | Description
-----|------------
-  `tag` | Define your own tag text. This will replace all other parameters
-  `grouping` | Is used to keep your tags organised under one 'folder'. Defaults to 'builds'
-  `prefix` | Anything you want to put in front of the version number (e.g. 'v')
-  `build_number` | The build number. Defaults to the result of increment_build_number if you're using it
-  `message` | The tag message. Defaults to the tag's name
-  `commit` | The commit or object where the tag will be set. Defaults to the current HEAD
-  `force` | Force adding the tag
-  `sign` | Make a GPG-signed tag, using the default e-mail address's key
-
-
+Key | Description | Default
+----|-------------|--------
+  `tag` | Define your own tag text. This will replace all other parameters | -
+  `grouping` | Is used to keep your tags organised under one 'folder'. Defaults to 'builds' | builds
```

Comment:
Test like "Defaults to 'builds'" should probably be removed then from the description, right?